### ISSUE TITLE
Add 'ignore-case' option for username comparison

### ISSUE THREAD

**Issue by stefan-rademacher-tk** [github/saml-ldap-team-sync#28]
Hi,

in many cases, our GitHub and LDAP users differ in case sensitivity.
It would be great to have an "--ignore-case" option, which replaces the lines
`add_users = list(set(ad_members) - set(ghe_members))`
and
`remove_users = list(set(ghe_members) - set(ad_members))`
by a more complex implementation.

Can you imagine to have this feature added to the script?

I implemented a quick and dirty function, which does the job (couldn't come up with a better function name):
```
def get_substract_list(list1, list2):
    result=[]
    lower_list2 = list(set(i.lower() for i in list2))
    for l in set(list1):
        lower_l = l.lower()
        if lower_l not in lower_list2:
            result.append(l)
    return result
```
First, this should be configurable via an option, and '-i' (like 'ignore case' in the grep command) is already used with different semantics. 
Second, I would rather have methods like `get_all_group_members()` and `get_new_group_members()` (replacing `get_group_members()`) in class `ADSync`, instead of my utility method. This would require a bigger refactoring, which I don't have the time to.

What do you think? I could open a pull request with my 'quick and dirty' solution. Which option name do you propose?

**Comment by primetheus** [comment]
@stefan-rademacher-tk thanks for opening this up! I think a bit of refactoring may be in order, but this will definitely help folks get moving in the short term. I'll take a look at the refactoring this week

**Comment by primetheus** [comment]
@stefan-rademacher-tk This is now implemented in #33, which will function as a GitHub App and introduce a lot of new functionality

**Comment by primetheus** [comment]
This is present now in the latest release
